src/mapml/layers/MapLayer.js

```diff
@@ -494,6 +494,18 @@ export var MapMLLayer = L.Layer.extend({
         opacityControlSummaryLabel = document.createElement('label'),
         mapEl = this._layerEl.parentNode;
 
+        let removeButton = document.createElement('a');
+        removeButton.href = '#';
+        removeButton.role = 'button';
+        removeButton.title = 'Remove Layer';
+        removeButton.innerHTML = '&#10006;';
```

Comment:
This `✖` character should be hidden from ATs (same as https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/241/). NVDA announces <q>multiply</q> on hover and <q>multiply remove layer, button</q> on keyboard focus. (This should also be done for characters in `.mapml-popup-button`.)